src/appshell/qml/Audacity/AppShell/aboutmodel.cpp

```
@@ -91,6 +321,25 @@ void AboutModel::toggleDevMode()
     globalConfiguration()->setDevModeEnabled(!globalConfiguration()->devModeEnabled());
 }
 
+QVariantList AboutModel::creditList() const
```

Cppcheck (2.20.0)

[style] The function 'creditList' is never used.

(unusedFunction)